tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,48 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(5)
+  require(shinytest2)
+
+  data <- simple_cdisc_data()
```

Comment:
can you prefix `simple_cdisc_data` ? does it come from tmg or any other package?

Reply:
It's a test helper

Reply:
this is helper function defined in  helper-TealAppDriver.R